pkgs/tools/video/gopro-tool/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, ffmpeg, vlc, jq }:
+
+stdenv.mkDerivation {
+  pname = "gopro-tool";
+  version = "unstable-2024-04-18";
+
+  src = fetchFromGitHub {
+    owner = "juchem";
+    repo = "gopro-tool";
+    rev = "a678f0ea65e24dca9b8d848b245bd2d487d3c8ca";
+    sha256 = "0sh3s38m17pci24x4kdlmlhn0gwgm28aaa6p7qs16wysk0q0h6wz";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/gopro-tool $out/bin/gopro-tool
+    chmod +x $out/bin/gopro-tool
+
+    wrapProgram $out/bin/gopro-tool \
+      --prefix PATH : ${lib.makeBinPath [ ffmpeg vlc jq ]}
+
+    cat <<'EOF' > $out/README
+GoPro Tool requires the following manual setup:
+
+1. **Enable the \`v4l2loopback\` kernel module** (needed for webcam support).
+   If you're on NixOS, add this to \`/etc/nixos/configuration.nix\`:
+   
+   ```nix
+   boot.extraModulePackages = with config.boot.kernelPackages; [ v4l2loopback ];
+   boot.kernelModules = [ "v4l2loopback" ];
+   ```
+
+   Then rebuild with:
+   ```sh
+   sudo nixos-rebuild switch
+   ```
+
+2. **Ensure your user is in the \`video\` group** to access \`/dev/video*\`:
+   ```sh
+   sudo usermod -aG video \$USER
+   ```
+
+   Then log out and log back in for the changes to take effect.
+
+3. **Allow UDP traffic on port 8554 in your firewall**:
+   ```nix
+   networking.firewall.allowedUDPPorts = [ 8554 ];
+   ```
+
+4. **Start the GoPro webcam stream**:
+   ```sh
+   gopro-tool start device
+   ```
+
+EOF
```

Comment:
Unnecessary to ship the README like this.

Reply:
I took this out. Where should I let people know they need the firewall port open, the v4l2loopback module etc?

Reply:
They should read the documentation from the package's website.